tests/drivers/uart/uart_async_api/boards/nucleo_l552ze_q.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&usart3 {
```

Comment:
Why adding usart3 in the overlay of the test ?
Why not adding usart3 in board Devicetree ?

Reply:
Change is limited to _that_ test only

Reply:
Since there is quite little devices specified in board description, it would indeed make sense to take advantage of this to add one spi device and one uart device to the boards (specially if these are arduino ones)